hibernate-rx-core/src/main/java/org/hibernate/rx/persister/entity/impl/IdentifierGeneration.java

```diff
@@ -81,15 +83,22 @@ static Properties identifierGeneratorProperties(
 				return new TableRxIdentifierGenerator(persistentClass, creationContext);
 			}
 			else if (structure instanceof SequenceStructure) {
-				return new SequenceRxIdentifierGenerator(persistentClass, creationContext);
+				return new SequenceRxIdentifierGenerator(
+						persistentClass,
+						creationContext,
+						createRxOptimizer(
+								((SequenceStyleGenerator) identifierGenerator).getOptimizer()
+						)
+				);
 			}
 			throw new IllegalStateException("unknown structure type");
 		}
 		else if (identifierGenerator instanceof TableGenerator) {
 			return new TableRxIdentifierGenerator(persistentClass, creationContext);
 		}
 		else if (identifierGenerator instanceof SequenceGenerator) {
-			return new SequenceRxIdentifierGenerator(persistentClass, creationContext);
+			throw new UnsupportedOperationException(
+					identifierGenerator.getClass().getName() + " is no longer supported. Use " + SequenceStyleGenerator.class.getName() + " instead.");
```

Comment:
{{SequenceGenerator}} is deprecated. Do we really need to support it here? If so, I'll implement as appropriate.

Reply:
I'm fine with not implementing it